node/scripts/build-packages.sh

```diff
@@ -4,7 +4,7 @@ if [[ -z ${RELEASE_VERSION+x} ]] || [[ -z ${DEVELOPMENT_VERSION+x} ]]; then
 fi
 
 AGENT_VERSION="7.55.3"
-TRACER_VERSION="4.45.0"
+TRACER_VERSION="4.55.0"
```

Comment:
Why is v4.x (which is deprecated since last week) still used here?

Reply:
This was to allow for support of Node 16 Azure App Service Web apps but it looks like Node 16 is deprecated. We can upgrade to use the v5.x release line.